pull.go

```diff
@@ -127,22 +148,29 @@ func (c *Client) Pull(ctx context.Context, ref string, opts ...RemoteOpt) (_ Ima
 	// download).
 	var ur unpack.Result
 	if unpacker != nil {
+		_, unpackSpan := tracing.StartSpan(ctx, "pull.UnpackWait")
 		if ur, err = unpacker.Wait(); err != nil {
```

Comment:
We probably also want a span within the unpack operation.  The pull.UnpackWait span will capture the delay imposed on the overall pull operation, but won't capture the actual time it takes to unpack a layer.